contrib/gitian-descriptors/gitian-guix-linux-win.yml

```diff
@@ -0,0 +1,98 @@
+---
+name: "bitcoin-core-guix-linux-win"
+enable_cache: true
+sudo: true
+distro: "ubuntu"
+suites:
+- "bionic"
+architectures:
+- "amd64"
+packages:
+- "ca-certificates"
+- "curl"
+- "gnupg"
+- "netbase"
+- "wget"
+remotes:
+- "url": "https://github.com/bitcoin/bitcoin.git"
+  "dir": "bitcoin"
+files: []
+script: |
+  set -e -o pipefail
+
+  export TZ="UTC"
+  export BUILD_DIR="$PWD"
+  if test -n "$GBUILD_CACHE_ENABLED"; then
+    export SOURCES_PATH=${GBUILD_COMMON_CACHE}
+    export BASE_CACHE=${GBUILD_PACKAGE_CACHE}
+    mkdir -p "${BASE_CACHE}" "${SOURCES_PATH}"
+    GUIX_CACHE=${GBUILD_COMMON_CACHE}/guix
+
+    GUIX_SCRIPT=${GUIX_CACHE}/guix-install.sh
+    if [[ -f "${GUIX_SCRIPT}" ]]; then
+      cp ${GUIX_SCRIPT} $BUILD_DIR
+    fi
+    GUIX_TARBALL=${GUIX_CACHE}/guix-tarball-pack-x86_64-linux.tar.gz
+    if [[ -f "${GUIX_TARBALL}" ]]; then
+      export GUIX_BINARY_FILE_NAME=${GUIX_TARBALL}
+    fi
+  fi
+  export JOBS=${NUM_PROCS}
+
+  # Install guix with the installer script
+  wget -nc https://git.savannah.gnu.org/cgit/guix.git/plain/etc/guix-install.sh
```

Comment:
Can we verify the SHA256 of this? (and peg it to the current version)

Reply:
Done